docs/cues/cue-playback.md

```diff
@@ -176,38 +176,57 @@ Settings](playback-options.md#fader-tab) in playback [Options](playback-options.
+
+You can set a mask so that some attributes will automatically release from the playback when the playback is killed. See [Release Mask](#release-mask) below.
+
+You can release a running playback by pressing
 \<Release\> *(\<Off\> button on Pearl Expert)*, then the **Select** button of
-the playback to be released. Channels will release back to their state
-in the previous playback until no playbacks are left to be released. If
+the playback to be released. Attributes will go back to the state
+set in the most recent previous playback until no playbacks are left to be released. If
 the \[Release to Home\] setting on the Release menu is Enabled, channels
 will go to the power-on state, which you can program (see 
-[next section](#programming-the-release-power-on-state)).
+[Programming the Release / Power On State](#programming-the-release-power-on-state)).
 If the option is disabled, channels will remain in the state of the last
-playback. You can release all running playbacks by pressing \<Release\>
-then \[Release all playbacks\].
+playback.
 
 You can also configure the key profile so that the blue or grey buttons
 can be used to release the playback using [Key Profiles](../system-settings/key-profiles.md).
 
 Channels will always release with a fade time, this defaults to 2
 seconds but you can change it using \[Master Release Time\] in the
 Release menu. You can also set an individual release time for the
-playback from the [Release tab](playback-options.md#release-tab) of playback [Options](playback-options.md).
+playback from the [Release tab](playback-options.md#release-tab) of playback options.
 
 -   You can instantly kill a playback by pressing \<Avo\> + the **Select**
     button of the playback.
 
--   You can set a mask so that only certain attributes release from the
-    playback when the playback is killed. This uses the 
-    [\[Release Mask\] option](playback-options.md#release-mask) in 
-    playback [Options](playback-options.md).
-    Normally all attributes will remain in the state last used in the
-    playback - *for cues which trigger strobing or other effects this
-    might not be what you want*.
+-   You can **Release All** running playbacks by double pressing \<Release\>. The user setting [Release Priority](../system-settings/user-settings.md#release) sets which playback priority levels will be released. You can protect against unintended Release All by setting the Release Priority to Low - this would only release playbacks with Low priority, if you had any set up.
+
+
+Release Mask
+------------
+
+You can configure which attributes will release from a playback when it is killed using the **Release Mask**. There is a Global
```

Comment:
Might be worth clarifying the distinction between the release mask set in the `<Release>` menu (a.k.a. the temporary release mask) and the release mask used for kill, set Globally / On a playback, as they're different things

Reply:
Can you clarify the temporary release mask ... As far as I know <Release> [Global Release Mask] sets the global mask for when playbacks are killed, or you can set a per-playback mask using Options for that playback - how do you set the temporary mask and what does it do?

Reply:
Correct, so the global/per-playback masks are for what type of release happens when the playback gets "killed", which variously happens when the user/timeline sets a playback level to zero, a fade out completes, or a user does Avo+Select (there are a number of playback options which modulate exactly when a playback gets killed out of the above options depending on the type of playback).

Avo+Select requests a kill directly, which will set the playback level to zero immediately:
* For memories, the playback fade out is triggered by setting the playback level to zero. The kill happens when that fade completes (or immediately on setting to zero, depending on the new user setting as discussed in #324).
* As cuelists/chases do not have playback fade times, the playback level hits zero instantly, which results in HTP levels being mastered to zero immediately. The kill occurs, leaving LTP data to release according to the mask & time.

Separately from a release triggered by a kill, you can use the Release menu to ask Titan to release a playback with specific settings (or to "release" a number of other things such as masters). You don't have to hold down `<Release>`; basically my understanding of how this menu operates is as follows:

1. Optional: you enter an override release time using the numeric keypad, putting you in the generic numeric entry menu.
1. You press `<Release>`. You are now in the "Release Playback Menu".
1. The attribute group/bank buttons are now indicating the temporary mask, which defaults to all groups. You can toggle the temporary mask with the group/playback buttons and `<Options>` as per usual.
1. Now, you may either:
    * Select a playback to cause it to be released over the master release time (or override release time if one was set), using the current temporary mask.
    * Select a timeline to cause it to be "released" (made inactive and level set to zero). The temporary mask has no effect.
    * Select a master/group to cause it to be "released" (level set so as to have no effect). The temporary mask has no effect.
    * Select `<Release>` again or the `{Release All Playbacks By Priority}` softkey to release all playbacks by priority (this is how `<Release><Release>` works). This should respect the current temporary mask and override / master release time (but see \*).
    * Select `<Clear>` to release the programmer over the master release time or temporary release time.
    * Select `<Connect>` to release the currently connected master.
    * Select a wheel `@` button to cause that attribute to be released from the programmer for the selected fixtures.
    * Select `Release All Masters` to reset all master levels. This does not use the temporary mask in any way.
    * Select `{Global Release Mask}`, which takes you to a menu which allows you to set the global release mask user setting. This is unrelated to the temporary mask.
    * Select a handle group `Go Page` button, which takes you the "Release Playbacks on Faders" menu for that handle group (With options `{Release Playbacks Not On This Page}`, `{Release This Page}`and `{Release All Playbacks In This Group}`. This should respect the current temporary mask (but see \*)

\* Trying this, it seems there is a bug 🐛: the temporary release mask is disregarded, and if the user has changed the temporary mask from All, then the override/master release time seems to be disregarded too. This is... unfortunate.